dom.bs

```diff
@@ -4363,6 +4363,28 @@ dom-Range-extractContents, dom-Range-cloneContents -->
  <a for=tree>children</a> of <var>node</var> and append them to <var>copy</var>, with
  <var>document</var> as specified and the <i>clone children flag</i> being set.
 
+ <li>If <var>node</var> is a <a for=Element>shadow host</a> whose <a for=/>shadow root</a>'s
+ <a for=ShadowRoot>clonable</a> flag is set, run these steps:
```

Comment:
```suggestion
 <a for=ShadowRoot>clonable</a> is true:
```
Right?

Also, needs to use `<p>` on a newline as the `<li>` has multiple children.